sdk/servicebus/azure-servicebus/tests/stress_tests/test_stress_queues.py

```diff
@@ -299,3 +301,50 @@ def test_stress_queue_close_and_reopen(self, servicebus_namespace_connection_str
         result = stress_test.run()
         assert(result.total_sent > 0)
         assert(result.total_received > 0)
+
+    class DroppedMessageCheckerStressTestRunner(StressTestRunner):
```

Comment:
hmmm, can you explain a bit what this stress test is doing?

Reply:
Yup!  (And I assume you mean in comment too so I can certainly do that, but for progeny here: ) It's a check motivated by Harsha having observed that over a million messages they observed some get lost due to <problems> on their end, and it seemed like (given our having users with this concern in EH) a reasonable thing to have validation for.

Reply:
```suggestion
    # This test validates that all individual messages are received contiguously over a long time period.
    # (e.g. not dropped for whatever reason, not sent, or not received)
    class DroppedMessageCheckerStressTestRunner(StressTestRunner):
```